ops/ansible/roles/bfd-pipeline/defaults/main.yml

```diff
@@ -8,6 +8,26 @@ data_pipeline_idempotency_required: true
 data_pipeline_new_relic_metric_host: 'https://gov-metric-api.newrelic.com'
 data_pipeline_new_relic_metric_path: '/metric/v1'
 
+## RDA API Pipeline
+# Used to download FISS/MCS pre-adjudicated claims data.
+#
+# Set to true to enable the RDA pipeline jobs.
+data_pipeline_rda_job_enabled: false
+
+# Communication settings for RDA API server.
+# These are defined in the vault rather than plain text.
+data_pipeline_rda_grpc_host: undefined
+data_pipeline_rda_grpc_port: 443
+
+# Interval at which the PipelineManager should ensure the jobs are running.
+data_pipeline_rda_job_interval_seconds: 300
```

Comment:
@brianburton it looks like there are over 16k FISS claims coming in from the RDA API on DEV and probably a similar amount of MCS.  I think we should dial this back a little, maybe have it run once an hour instead of every 5 mins.  At least until we build in support for replays.  So maybe just leave this as is in this file and in make it `3600` in `test/group_vars/all/env_specific.yml`

Reply:
This is where the code I had in place originally to allow us to configure a limit on the number of records the job would write per scheduled execution would have helped.  It was removed based on reviewer comments though.

OK.  I will increase the interval as requested.  Generally speaking the value of this isn't all that important for our job since the job will run forever if the idle time is never exceeded.  If the server call fails the PipelineManager won't restart us anyway.

Reply:
The 3600 value has been added in `test/group_vars/all/env_specific.yml` now.

Reply:
Yeah I was assuming that we'd address the restart issues in the near future.  Maybe its good to run through a scenario real quick to see how the settings will work.

From the DB, it looks like we consumed the ~16k FISS claims in about 9 mins.  When we first connect to RDA, if we spend 9 mins consuming data, then disconnect after an additional 10 mins (the idle time).  So we'd have roughly 40 mins or so (assuming the above 3600 interval value), before we repeated the process all over again.  Does that sound right?

Reply:
Yes I think that's correct.